modules/images-other-jenkins-agent-env-var.adoc

```diff
@@ -50,4 +50,8 @@ JVM threads.
 |Specifies additional options for the Jenkins JVM. These options are appended to all other options, including the Java options above, and can be used to override any of them, if necessary. Separate each additional option with a space and if any option contains space characters, escape them with a backslash.
 |Example settings: `-Dfoo -Dbar`; `-Dfoo=first\ value -Dbar=second\ value`
 
+|`USE_JAVA_VERSION`
+|Specifies the version of Java to inject into the container agent. The default value is `java-11`.
```

Comment:
The more exact wording would be "Specifies the version of Java version to use to run the agent in its container. The default value is `java-11`. Possible alternative value is `java-1.8.0`. If you extend the container base image, you can specify any alternatives version of java using it associated suffix".

@rolfedh it follows the pattern explained here 
https://access.redhat.com/documentation/en-us/openjdk/8/html-single/installing_and_using_openjdk_8_for_rhel/index